OpenRA.Utility/Program.cs

```diff
@@ -28,7 +28,6 @@ static void Main(string[] args)
 			AppDomain.CurrentDomain.AssemblyResolve += GlobalFileSystem.ResolveAssembly;
 
 			Log.AddChannel("perf", null);
-			Log.AddChannel("debug", null);
```

Comment:
But ReplayMetadata and ReplayReport write error messages to debug log. Shouldn't the utility use that?